modules/vaos/app/serializers/vaos/institution_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module VAOS
+  class InstitutionSerializer
+    include FastJsonapi::ObjectSerializer
+
+    set_id :institution_ien
```

Comment:
I don't think this will always be unique, if that matters here. `location_ien` should be.